src/Mailjet/Client.php

```diff
@@ -151,7 +151,7 @@ private function _setTokenAuthetication($token, $call, $settings)
      * @return boolean flag
      */
     private function _isBasicAuthenticationRequired($key, $secret)
-    {        
+    {
         return is_string($key) && is_string($secret) && empty($secret);
```

Comment:
That line is also wrong. It should be 
```php
return is_string($key) && is_string($secret) && !empty($secret);
```

Reply:
Not really in the scope of this PR but I can still add this fix :}

Reply:
Mind doing a suggestion ?

Reply:
:) Just add a "!" in front of `empty($secret)`

Reply:
I don't have my pc on me right now, so it would have been easier for me to just accept the suggestion, it's a new feature from github. :}

I'll try to get some time to add it in the afternoon otherwise.

```suggestion
        return is_string($key) && is_string($secret) && !empty($secret);
```

Reply:
ehh it seems I can suggest myself. Done then. :}

Reply:
Sweet ! Sorry, I didn't know that feature !